changelogs/current.yaml

```diff
@@ -68,6 +68,10 @@ bug_fixes:
   change: |
     Fixed an issue using the cluster wide CONNECT termination so it will successfully proxy payloads.
 
+- area: upstream
+  change: |
+    Fixed the LOGICAL_DNS cluster to work for IPv6.
```

Comment:
We have same issue for the strict DNS https://github.com/envoyproxy/envoy/blob/e8ab7628467187ece59d9722d4cee7eb23c0eee3/source/common/upstream/strict_dns_cluster.cc#L92

Reply:
I think you were looking at the first commit - I already did both those changes in the last push :-)

Reply:
hah, you are right,sorry about that, then this LGTM :)